_posts/blog/2019-06-23-About-Georgia.md

```diff
@@ -0,0 +1,77 @@
+---
+layout: post
+title: About Georgia
+categories: blog
+excerpt: Researcher in Kirstie's lab working on citizen science
+tags: [onboarding, introducing]
+image:
+  feature:
+link:
+date: 2019-06-14
+modified:
+share: true
+author: GeorgiaHCA
+---
+
+## What I do at the Turing
+
+I joined the Turing Institute for a 2-year contract, based in Kirstie’s lab, working on participatory science, sensory processing, and autism. My research interests include data ethics and the philosophy of technology. I’m currently building a citizen science platform and developing a participatory science framework.
+
+Before joining the Turing I was a civil servant, and I studied English literature for my undergrad and M.Phil degrees.
+
+I am open to collaborating and am interested in helping out with projects and papers – so get in touch if you’d like to join forces!
+
+## Where I'm From
+
+<figure>
+  <img src="/images/Lauder.jpg"
+       alt="Lauder">
+</figure>
+
+I’m from a tiny town in the Scottish Borders called Lauder. There aren’t that many people there, but there’s lots of trees, sheep, and rain. If you like rolling emerald hills and can tolerate numberless midges and never-ending mud, I’d recommend a visit!
+
+Here I am as a tiny kid (the horse’s name is Godiva <3) – I was too young to form any memories of this moment, so I don’t know if I was happy or terrified :p
+<figure>
+  <img src="/images/Godiva.jpg"
+       alt="Godiva">
+</figure>
+
+I’ve spent most of my adult life in London, but I’ve also lived in Oxford (where I did my undergrad) and Cambridge (for my M.Phil), as well as in Paris and Beijing. I can tell I’ve lost my Scottish stoicism because I like my porridge with hot cream and strawberries instead of cold water. Still undecided on whether I prefer a dry martini, dry socks, and dry humour to a hot toddy, damp boots, and Braveheart references though.
+
+I have a twin sister who's a professional pianist in Edinburgh. Like all twins, we share not only genes but dreams, and we have telepathic powers. Here she is playing:
+
+<figure>
+  <img src="/Images/AilsaPiano.jpg"
+       alt="Ailsa">
+</figure>
+
+## What I Like and Motivations
+
+Music: I play saxophone and I love jazz, especially Charlie Parker. I could easily go crazy listening to Crazeology on loop forever if friends didn't intervene.
+Running: Just for the endorphins! I’m not especially disciplined or competitive.
+Reading!!!!!!! I'm a greedy omnivore who appreciates both books and computers.
+
+<figure>
+  <img src="/Images/ReadWhileYouRun.jpg">
+  <figcaption> This looks dangerous - full disclosure I've never risked this level of efficiency! Image comes from http://www.saltyrunning.com/readers-roundtable-read-run/ </figcaption>.
```

Comment:
This url doesn't render in the blog post. Can you try making it an html formatted one?

```suggestion
  <figcaption> This looks dangerous - full disclosure I've never risked this level of efficiency! Image source <a href="http://www.saltyrunning.com/readers-roundtable-read-run/">here</a>.</figcaption>.
```